### DISCUSSION TITLE
Make Revolt More Than A Chat App

### DISCUSSION THREAD

**Discussion by ghost** [stoatchat/discussions#669] [Feature Suggestions]
We could expant revolt.chat to integrate more useful functions such as video sharing, social networking.

I feel like if revolt.chat can implement stuff like this in the app it would really help it take off I know it's ambitious and very difficult but there is alot of talented programmers who can contribute to make this hsppen

**Comment by YukitoNakamura** [comment]
I'd also add Forum and Announcement channels like Guilded, although they would be easier to work with than full social networking.

**Reply by ghost** [reply]
There are alot of talented programmers therefore this is a realistic option especially since its open source

**Comment by Rexogamer** [comment]
We're currently busy with the new frontend, which will be followed by the new voice server. We are also rebuilding the internal admin panel to make moderation easier, and working on native mobile apps.

I don't think social networking or YouTube-style video sharing are particularly relevant to our aims for the platform, and we are busy enough with the features we intend to add/have put on the roadmap.

I also want to re-iterate something that @insertish has stated previously - a lot of people have said that there are plenty of devs willing to help... but we don't see many PRs? A lot of the work is still done solely by Insert and almost everything is done by the core team. There may be room for us to improve the contribution process/make things easier to merge, but at some point the question has to be asked - **where are these often-mentioned contributors?**

**Reply by SnenxyTengoku** [reply]
Yesss..... yess...... remove the cla.... yes..... .goood......

**Reply by lo2dev** [reply]
>where are these often-mentioned contributors

i eated them, soz,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

**Reply by ghost** [reply]
It was only a suggestion also Im not apart of the community therefore Im unable to find contributors, Im still struggling with learning the basics of css and JavaScript só i Would not feel comfortable contributing at the moment
